tabbycat/adjfeedback/models.py

```diff
@@ -256,7 +256,7 @@ def __str__(self):
 
     def _unique_unconfirm_args(self):
         kwargs = super()._unique_unconfirm_args()
-        if self.source_team is not None and self.source_team.debate.round.tournament.pref('feedback_from_teams') != 'all-adjs':
+        if self.source_team is not None and self.source_team.debate.round.tournament.pref('feedback_from_teams') == 'orallist':
```

Comment:
Not quite sure about this... if the setting is for no one, then we shouldn't even reach here as no options should appear.